packages/docid/src/index.ts

```diff
@@ -209,11 +195,17 @@ class DocID {
   get bytes (): Uint8Array {
     if (this._bytes == null) {
       const codec = varint.encode(DOCID_CODEC)
-      const subnet = varint.encode(this._subnet)
-      const doctype = varint.encode(this._doctype)
-      const versionBytes =  this._version ? this._version.bytes : new Uint8Array(0)
+      const doctype = varint.encode(this.type)
+
+      let versionBytes 
+      if (this.version) {
+        versionBytes = this.cid.equals(this.version) ? varint.encode(0) : this.version.bytes
```

Comment:
I guess you could just do `new Uint8Array([0])` instead of using varint.

Reply:
yeah, just used for clarity, but i guess it will never be any other int, so could be